docs/integrations/sources/xero-migrations.md

```diff
@@ -0,0 +1,25 @@
+# Xero Migration Guide
+
+## Upgrading to 1.0.0
+
+The authentication scheme is now using `access_token` instead of Oauth 2.0, Follow the documentation - https://developer.xero.com/documentation/guides/oauth2/pkce-flow for understanding more about how to get access token.
```

Comment:
```suggestion
The authentication schema is now using `access_token` instead of Oauth 2.0. Visit the Xero documentation - https://developer.xero.com/documentation/guides/oauth2/pkce-flow for more detailed information about how to get access token.
```